src/mixpanelMiddleware.js

```diff
@@ -46,6 +47,10 @@ export default function mixpanelMiddleware(token, options = {}) {
             });
         }
 
+        if (timeEvent && timeEvent.length > 0) {
+            mixpanel.time_event(timeEvent);
```

Comment:
[API docs](https://mixpanel.com/help/reference/javascript-full-api-reference#mixpanel.time_event) mention only one param event_name. Can you point me to right direction so I can understand it?